adapt.go

```diff
@@ -39,6 +40,10 @@ func (bs BlockStep) opaqueCursor() string {
 	return bs.cursor
 }
 
+func (bs BlockStep) previousOpaqueCursor() string {
```

Comment:
Is that an opaque cursor or a cursor that you send ?

Reply:
Opaque cursor is the serialized form of a cursor object and it's used as an header